curvefs/src/metaserver/inode_manager.cpp

```diff
@@ -248,6 +236,8 @@ MetaStatusCode InodeManager::UpdateInode(const UpdateInodeRequest &request) {
     UPDATE_INODE(gid)
     UPDATE_INODE(mode)
 
+    *(old->mutable_parent()) = request.parent();
```

Comment:
if (request.has_parent())?

Reply:
The parent is a 'repeated' type has no 'has_*' and the parent field is Inherent property of inode like uid,gid,mode. So here can check parent_size(), but feel unnecessary.